xblock/core.py

```diff
@@ -205,6 +205,28 @@ def __init__(self, runtime, field_data=None, scope_ids=UNSET, *args, **kwargs):
         # Provide backwards compatibility for external access through _field_data
         super().__init__(runtime=runtime, scope_ids=scope_ids, field_data=field_data, *args, **kwargs)
 
+    @property
+    def usage_key(self):
+        """
+        Convenient abbreviation for `XBlock.scope_ids.usage_id`.
+        """
+        return self.scope_ids.usage_id
+
+    @property
+    def context_key(self):
+        """
+        Convenient abbreviation for `.XBlock.scope_ids.usage_id.context_key`.
```

Comment:
Nit: On the first line of the docstring, I would prefer to say something more helpful like "This is the ID of the context (course, library, etc.) that contains this XBlock". Then, you can mention that it's an alias.

Same above for `usage_key`/`usage_id` - what _is_ it?

Reply:
Good call, I'll make that change.